.github/workflows/docker-build.yml

```diff
@@ -47,13 +47,17 @@ jobs:
         #
         # Optional properties:
         # freeDiskSpace: Whether to free disk space before building the image.
+        # skipWorkerBaseImage: Whether to skip building the worker base image.
```

Comment:
Maybe same this positively, e.g. "buildWorkerBaseImage"?

Reply:
A general question: Why does this have to be modeled / optimized manually? Shouldn't the `:core:jibDockerBuild` task automatically skip building the base worker image if it the task is not depending on it?

Maybe you can detail this in the commit message more.